### PR TITLE
feat: Price Estimation contact information page

### PR DESCRIPTION
**Pull request by olerichter00**
Addresses [CX-2781]

## Description

Introduces the new Price Estimate contact information page (behind feature flag).

You can start your review [here](src/Apps/MyCollection/Routes/PriceEstimate/PriceEstimateContactInformation.tsx), this is the component for the new page. Sorry for all the other changes....

### Screenshots

Screenshot 2022-10-11 at 10 08 57

[CX-2781]: https://artsyproduct.atlassian.net/browse/CX-2781?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/Apps/Consign/Routes/SubmissionFlow/ContactInformation/Components/ContactInformationForm.tsx [hunk 1]
```diff
@@ -84,3 +84,23 @@ export const ContactInformationForm: React.FC<ContactInformationFormProps> = ({
     </Box>
   )
 }
+
+export const ContactInformationFormFragmentContainer = createFragmentContainer(
```
Comment (olerichter00) [inline src/Apps/Consign/Routes/SubmissionFlow/ContactInformation/Components/ContactInformationForm.tsx:88]: Adding a fragment container to use the form in the SWA flow and the Request Price Estimate flow.

File: src/Apps/MyCollection/Routes/MyCollectionArtwork/Components/MyCollectionArtworkRequestPriceEstimateSection.tsx [hunk 2]
```diff
@@ -0,0 +1,66 @@
+import { Button, Flex, Separator, Text, WinningBidIcon } from "@artsy/palette"
```
Comment (olerichter00) [inline src/Apps/MyCollection/Routes/MyCollectionArtwork/Components/MyCollectionArtworkRequestPriceEstimateSection.tsx:1]: I just corrected the name of the file, this is not a new component.

File: src/Apps/MyCollection/Routes/PriceEstimate/PriceEstimateContactInformation.tsx [hunk 3]
```diff
@@ -0,0 +1,189 @@
...
+    } catch (error) {
+      console.error(error)
+
+      sendToast({
+        variant: "error",
+        message: "An error occurred",
+        description: "Please contact sell@artsymail.com",
+      })
+
+      return
```
Comment (lordkiz) [inline src/Apps/MyCollection/Routes/PriceEstimate/PriceEstimateContactInformation.tsx:99]: **Non-Blocking**: `return` may be unnecessary here 🤔

File: src/Apps/MyCollection/Routes/PriceEstimate/Mutations/useRequestPriceEstimate.ts [hunk 4]
```diff
@@ -0,0 +1,28 @@
+import { graphql } from "react-relay"
+import { useMutation } from "Utils/Hooks/useMutation"
+import { useRequestPriceEstimateMutation } from "__generated__/useRequestPriceEstimateMutation.graphql"
+
+export const useRequestPriceEstimate = () => {
```
Comment (lordkiz) [inline src/Apps/MyCollection/Routes/PriceEstimate/Mutations/useRequestPriceEstimate.ts:5]: **Non-Blocking**: The name could be more descriptive. I think if it communicates that it is a mutation, it will be more helpful. Something like: `useMutationForRequestPriceEstimate` 🤷

File: src/Apps/MyCollection/Routes/PriceEstimate/__tests__/PriceEstimateContactInformation.jest.tsx [hunk 5]
```diff
@@ -0,0 +1,280 @@
...
+    expect(mockTrackEvent).toHaveBeenCalled()
+    expect(mockTrackEvent).toHaveBeenCalledWith({
+      action: ActionType.sentRequestPriceEstimate,
+      artwork_id: mockArtwork.internalID,
+      artwork_slug: "an-artwork",
+      user_id: "123",
+      user_email: "serge@test.test",
+    })
+  })
+})
```
Comment (lordkiz) [inline src/Apps/MyCollection/Routes/PriceEstimate/__tests__/PriceEstimateContactInformation.jest.tsx:280]: 👏 Well-thought out and nicely written tests. It reads like a book. Cooooool!

File: src/Apps/MyCollection/myCollectionRoutes.tsx [hunk 6]
```diff
@@ -41,11 +52,6 @@ export const myCollectionRoutes: AppRouteConfig[] = [
     onClientSideRender: () => {
       MyCollectionArtwork.preload()
     },
-    prepareVariables: ({ artworkID }) => {
-      return {
-        artworkID,
-      }
-    },
```
Comment (lordkiz) [inline src/Apps/MyCollection/myCollectionRoutes.tsx:48]: Why do we no longer need this?

**Review by lordkiz — APPROVED** [review]
Looks great to me. Left a few comments - non of them blocking.

File: src/Apps/MyCollection/myCollectionRoutes.tsx (hunk 6 above)
Comment (dariakoko) [inline src/Apps/MyCollection/myCollectionRoutes.tsx:48]: Why did you remove this? 👀

**Review by dariakoko — APPROVED** [review]
⭐
Looks good! 
I left only one non-blocking question

File: src/Apps/MyCollection/Routes/PriceEstimate/PriceEstimateContactInformation.tsx (hunk 3 above)
Comment (olerichter00) [reply src/Apps/MyCollection/Routes/PriceEstimate/PriceEstimateContactInformation.tsx:99]: That's true, I'll remove the return 👍

File: src/Apps/MyCollection/myCollectionRoutes.tsx (hunk 6 above)
Comment (olerichter00) [reply src/Apps/MyCollection/myCollectionRoutes.tsx:48]: It's automatically returning `artworkID`. This is only needed if the variables are manipulated. I thought removing it will make the file a bit smaller and easier to read.

Comment (olerichter00) [reply src/Apps/MyCollection/myCollectionRoutes.tsx:48]: It's automatically returning artworkID. This is only needed if the variables are manipulated. I thought removing it will make the file a bit smaller and easier to read.

**Review by lordkiz — APPROVED** [review]
